test

```diff
@@ -59,3 +59,4 @@ test coroutines-co.js
 test coroutines-bluebird.js
 test await.compiled.js
 test tasks.js
+test kgo.js
```

Comment:
Let's put this after __async.js__, both here and in the README. The idea is to group similar approaches.